Documentation/Books/Manual/Programs/Arangorestore/Examples.md

```diff
@@ -190,6 +202,28 @@ If you restore a collection that was dumped from a cluster into a single
 ArangoDB instance, the number of shards, replication factor and shard keys will silently
 be ignored.
 
+Note that in a cluster, every newly created collection will have a new
```

Comment:
this section here (next 3 rows) was removed in devel by this commit from other PR (https://github.com/arangodb/arangodb/commit/b3a3e35f3dfaf06ee6e896695ddc051857c5aa52#diff-57875d30de31f16d6048c4df9cba987d) - removing from here as well (part of conflict resolution)